layoutlm/scripts/funsd_preprocess.py

```diff
@@ -107,9 +107,9 @@ def seg_file(file_path, tokenizer, max_len):
             if current_subwords_len == 0:
                 continue
 
-            if (subword_len_counter + current_subwords_len) > max_len:
+            if (subword_len_counter + current_subwords_len) >= max_len:
```

Comment:
When the `(subword_len_counter + current_subwords_len) == max_len`, the new sequence will be correctly created in the next loop. So this is not the issue.

Reply:
Looking at the logic again, I think you're right but I was getting `Maximum sequence length exceeded` warning without the `==` check. Will have another look and see why that's the case.

Reply:
Removed the equality check, I think the problem is at somewhere else. Will report back when I found the issue.